test/test_integration_credentials_keyring.py

```diff
@@ -273,8 +276,8 @@ def test_https_logout_successful(self):
         self.assertIn('Successfully logged out from Conjur', output.strip())
```

Comment:
this isn't enough IMO. We want to validate that the entry in the keyring isnt there

Reply:
@mbenita-Cyberark